### PR TITLE
add stacked-backproject

### PR DESCRIPTION
**Pull request by harishkumar-harihara**
Reconstruct 2 or 4 slices in parallel

### CODE REVIEW

**Comment by tfarago** [comment]
Dear Harish, this already looks good! However, the `(ufo-launch:17644): GLib-GObject-CRITICAL **: 10:35:11.525: validate_pspec_to_install: assertion 'G_IS_PARAM_SPEC (pspec)' failed` is there because you declare 12 properties, but in `ufo_stacked_backproject_task_class_init` you only define 10. Also, could you please add the task and kernel files to meson (src/meson.build and src/kernels/meson.build)? And one last thing, please provide a description of what this is doing in the [docs](https://github.com/ufo-kit/ufo-filters/blob/master/docs/filters.rst).

**Comment by tfarago** [comment]
I already have some fixes, if you give me [push](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork) right to your repo we can fix the code together.

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -79,6 +79,7 @@ set(ufofilter_SRCS
     ufo-slice-task.c
     ufo-sliding-stack-task.c
     ufo-stack-task.c
+    ufo-stacked-backproject-task.c
```
[the remark this answers is not in the document]
Comment (harishkumar-harihara) [reply src/CMakeLists.txt:82]: Included the files in meson.build.

**Review by tfarago — APPROVED** [review]

**Comment by tfarago** [comment]
Nice work!

**Pull request merged**